metrotranscom/doorway
# feat: custom maps zoom controls

# Pull Request Template

## Issue Overview

This PR addresses [#issue](https://github.com/metrotranscom/doorway/issues/121)


## Description

Adds custom zoom controls to the /listings map

## How Can This Be Tested/Reviewed?

Go to the /listings page and see the map zoom controls. The `-` button is using the `+` icon until we can get an icon for `-`. Check that the zoom controls work as expected. 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
